copydb/copydb.go

```diff
@@ -21,6 +21,12 @@ func NewFerry(config *Config) *CopydbFerry {
 		Config: config.Config,
 	}
 
+	ferry.ErrorHandler = &ghostferry.PanicErrorHandler{
+		Ferry:             ferry,
+		ErrorCallback:     config.ErrorCallback,
+		DumpStateToStdout: false,
+	}
```

Comment:
if the `ErrorCallback` config is in the ghostferry/ferry.go file, and the only place where we use it in the copydb package?  This doesn't seem right. If an error callback is provided to the base ghostferry library, it should just use it.